python/libcudf/CMakeLists.txt

```diff
@@ -55,3 +62,14 @@ if(USE_NVCOMP_RUNTIME_WHEEL)
     APPEND
   )
 endif()
+
+# TODO(nvcomp): when `nvcomp` supports Python 3.13 and we de-vendor `nvcomp` from `kvikio` this
+# block should be removed
+if(USE_NVCOMP_FROM_LIBKVIKIO_WHEEL)
```

Comment:
Just putting this in writing to make it explicit... I think it's ok to allow `USE_NVCOMP_RUNTIME_WHEEL` and and `USE_NVCOMP_FROM_LIBVIKIO_WHEEL` to both be `ON`, as you've done here.

Having both RPATHs set on `libcudf.so`, even if accidentally, shouldn't be a problem. But just calling out that possibility, to give you an opportunity to say "oh yeah, I hadn't thought of that... ok no that's bad, let's make them mutually exclusive".

Reply:
That's a good callout -- I do think that it's ok if they're both set.  It should be a temporary addition, in any case.